appservice/src/startStreamingLogs.ts

```diff
@@ -79,8 +82,13 @@ export async function startStreamingLogs(client: SiteClient, verifyLoggingEnable
                         outputChannel: outputChannel
                     };
 
-                    logsRequest.on('data', (chunk: Buffer | string) => {
-                        outputChannel.appendLine(chunk.toString());
+                    logsRequest.on('data', (data: Buffer | string) => {
```

Comment:
What exactly is causing the bug?  Needs more commehnts.

Reply:
Added more comments

Reply:
So how likely is it that data will come in right before the tick and the duplicated data right after the tick?